src/components/checkbox/Checkbox/checkbox.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../../styles/theme";
+
+.checkbox-wrapper {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  .text {
+    margin-left: $spacing_0_8;
+    font-size: $font_size_1_4;
+    font-weight: $font_medium;
+    color: &color_black_37;
+  }
+
+  & > input {
+    -webkit-appearance: none;
+    appearance: none;
+
+    width: 2rem;
+    height: 2rem;
+    border-radius: 0.5rem;
+    border: $color_gray_92989b 0.2rem solid;
+
+    &:hover {
+      border: $color_secondary_9749fa 0.2rem solid;
```

Comment:
Following the design, the width of the border should be 1.5px which is 0.15rem. Can you please check that?